Resources/Prototypes/Catalog/Fills/Boxes/medical.yml

```diff
@@ -126,3 +126,21 @@
     whitelist:
       tags:
         - BodyBag
+
+- type: entity
+  name: vial box
+  parent: BoxCardboard
+  id: BoxVial
+  description: A box full of vials.
+  components:
+  - type: Storage
+    grid:
+    - 0,0,2,1
+  - type: StorageFill
+    contents:
+      - id: BaseChemistryEmptyVial
+        amount: 3
```

Comment:
make it 6 and also consider adding those little beaker stands. They'd just be whitelist storage that has itemmapper for displaying the fills.